Bio/AlignIO/MafIO.py

```diff
@@ -625,12 +696,16 @@ def get_spliced(self, starts, ends, strand=1):
                             if ref_first_strand not in (1, -1):
                                 raise ValueError("Strand must be 1 or -1")
                         elif ref_first_strand != seqrec.annotations["strand"]:
-                            raise ValueError("Encountered strand='%s' on target seqname, "
-                                             "expected '%s'" %
-                                             (seqrec.annotations["strand"], ref_first_strand))
+                            raise ValueError(
+                                "Encountered strand='%s' on target seqname, "
+                                "expected '%s'"
+                                % (seqrec.annotations["strand"], ref_first_strand)
+                            )
                     except KeyError:
```

Comment:
Same here, the Strings above could be joined in one line.

Reply:
This line would then be over 88 characters long.